pkg/operation/botanist/addons.go

```diff
@@ -226,6 +226,9 @@ func (b *Botanist) generateCoreAddonsChart(ctx context.Context) (*chartrenderer.
 			"vpaEnabled":        b.Shoot.WantsVerticalPodAutoscaler,
 		}
 		coreDNSConfig = map[string]interface{}{
+			"global": map[string]interface{}{
+				"nodeNetwork": b.Shoot.GetNodeNetwork(),
+			},
```

Comment:
Shouldn't this be here:

https://github.com/gardener/gardener/blob/fb1715be4030b69279ff75e2337a417d4e7e66ee/pkg/operation/botanist/addons.go#L223-L227

Reply:
Why? I wanna have it in the `coredns` chart and not in the `components` chart.

Reply:
Why? I wanna have it in the `coredns` chart and not in the `components` chart.

Reply:
You have added it to the `global` field in the `values.yaml`, see https://github.com/gardener/gardener/pull/3687/files#diff-e0e1ddf2d0b93e57689704cca6bffbba3f2022e115ea56defdea31d239aff3a1R6.

Reply:
Yes, for the `coredns` chart but not for the `components` chart, which would be here https://github.com/gardener/gardener/blob/master/charts/shoot-core/components/values.yaml#L1-L4

Reply:
OK, then it might work by accident. Still, I suggest to have it not below `global` if you only want it for CoreDNS (even if you can make it work like this). The `global` should only be used for values shared across multiple charts.

Reply:
I removed it from the global section.